Meta/ladybird.sh

```diff
@@ -134,6 +134,10 @@ build_target() {
 
 delete_target() {
     [ ! -d "$BUILD_DIR" ] || rm -rf "$BUILD_DIR"
+
+    # Delete the vcpkg user variables created by this script if they exist
+    VCPKG_USER_VARS="$LADYBIRD_SOURCE_DIR/Meta/CMake/vcpkg/user-variables.cmake"
+    [ ! -f "$VCPKG_USER_VARS" ] || rm "$VCPKG_USER_VARS"
```

Comment:
oh hm. should we drop these in the build dir like the `CC` and `CXX` variables are currently placed? and/or create the settings from the vcpkg-tuning script that creates that file?

Reply:
Moving the user-variables.cmake into the build dir would be the best option, IMO. I'm not that familiar with CMake but I can investigate. FWIW, the user-variables.cmake file is generated by ladybird.sh so I thought this was a reasonable place to do the cleanup.

Reply:
yeah for sure. I just am thinking about how to best clean this up. I added the user-variables.cmake hack before i added the one in the build dir currently. Let's go with this for now and leave moving files around to a follow-up

Reply:
oh, before we do that, can you fixup-squash your second commit into the first, and change the commit category to "Meta" instead of Build/CMake ? (the ladybird.sh script lives in Meta)

Reply:
Commits are squashed and I updated the category.